fitbenchmarking/fitbenchmarking
# Updating example table and plot in docs

#### Description of Work

Fixes #562 


#### Testing Instructions

1. Check new plot and table are consistent with new results pages

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
